From 7de94d74da3403a07a2460efb461484664201ec1 Mon Sep 17 00:00:00 2001
From: Mika Laitio <lamikr@pilppa.org>
Date: Mon, 24 Mar 2025 18:06:05 -0700
Subject: [PATCH 1/2] disable precompiled hip headers

- disable because precompiled headers can cause
  problems on gfx10 gpus
- todo: verify that nothing in openmpi does
  not try to use the disabled features.

Signed-off-by: Mika Laitio <lamikr@pilppa.org>
---
 hipamd/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hipamd/CMakeLists.txt b/hipamd/CMakeLists.txt
index 9538dbbad..356e2e4f8 100755
--- a/hipamd/CMakeLists.txt
+++ b/hipamd/CMakeLists.txt
@@ -43,7 +43,7 @@ list(APPEND CMAKE_MODULE_PATH ${HIP_COMMON_DIR}/cmake)
 #############################
 # Options
 #############################
-option(__HIP_ENABLE_PCH "Enable/Disable pre-compiled hip headers" ON)
+option(__HIP_ENABLE_PCH "Enable/Disable pre-compiled hip headers" OFF)
 option(HIP_OFFICIAL_BUILD "Enable/Disable for mainline/staging builds" ON)
 option(FILE_REORG_BACKWARD_COMPATIBILITY "Enable File Reorg with backward compatibility" OFF)
 option(BUILD_SHARED_LIBS "Build the shared library" ON)
-- 
2.41.3

